README.md

```diff
@@ -3,7 +3,7 @@
 Contributors: lukecarbis, ryankienstra, Stino11, rheinardkorf
 Tags: Gutenberg, Blocks
 Requires at least: 5.0
-Tested up to: 5.1.1
+Tested up to: 5.2.1
```

Comment:
This might be able to be simply `5.2`. That's how it is [on the AMP plugin](https://github.com/ampproject/amp-wp/blob/a042ea26011145a6ff790dc0b52697c0e813e5b4/readme.txt#L5)

Reply:
Might as well put the latest version, though, right?

Reply:
I think it might help to make this `5.2`, so we don't have to update it for WP `5.2.2`.

It looks like WordPress.org displays the latest point version as the "Tested up to" version.

For example, the AMP plugin's [Tested up to version](https://plugins.trac.wordpress.org/browser/amp/trunk/readme.txt#L50) is `5.2`. But WordPress.org displays it as `5.2.1`:

<img width="1011" alt="tested" src="https://user-images.githubusercontent.com/4063887/58148791-c27b3780-7c25-11e9-9842-c3f50634e379.png">